src/renderer/entities/proxy/api/pureProxiesService.ts

```diff
@@ -0,0 +1,14 @@
+import { GraphQLClient } from 'graphql-request';
+
+import { CHECK_PURE_PROXIES } from './graphql/queries/pureProxies';
+import type { AccountId } from '@shared/core';
+
+export const checkPureProxies = async (client: GraphQLClient, accountIds: AccountId[]): Promise<AccountId[]> => {
```

Comment:
Function name is misleading, should be renamed
It actually requests data and returns `ids` but name `check` tells us nothing
Something like: `getAccountIdsWithPureProxies` don't quite like it, can discuss other variants